zigpy/quirks/v2/__init__.py

```diff
@@ -539,6 +580,37 @@ def replaces(
         self.replaces_metadata.append(replace)
         return self
 
+    def replace_cluster_occurrences(
+        self,
+        replacement_cluster_class: type[Cluster | CustomCluster],
+        replace_server_instances: bool = True,
+        replace_client_instances: bool = True,
+    ) -> QuirkBuilder:
+        """Add a ReplaceClusterOccurrencesMetadata entry and returns self.
+
+        This method allows replacing a cluster on a device across all endpoints
+        for the specified cluster types when the quirk is applied.
+
+        replacement_cluster_class should be a subclass of Cluster or CustomCluster and
+        will be used to create a new cluster instance to replace the existing cluster.
+
+        If cluster_id is provided, it will be used as the cluster_id for the cluster to
+        be removed. If cluster_id is not provided, the cluster_id of the replacement
+        cluster will be used.
```

Comment:
`cluster_id` isn't a parameter for this. The text is copied from above, I think we can just remove this part? Don't see a use-case for this here at least.

```suggestion
```

Reply:
copy pasta error